src/initial_conditions/initial_conditions.jl

```diff
@@ -358,6 +358,101 @@ function (initial_condition::RisingThermalBubbleProfile)(params)
     return local_state
 end
 
+"""
+    RCEMIPIIProfile(; temperature = 300)
+
+An `InitialCondition` following the sounding to initialize simulations for
+RCEMIPII as described by Wing et. al. (2018). Note - this should be used
+for RCE_small and NOT RCE_large - RCE_large must be initialized with the
+final state of RCE_small. Temperature options are 295, 300, and 305.
+"""
+Base.@kwdef struct RCEMIPIIProfile{T} <: InitialCondition
+    temperature::T = 300
+end
+
+function (initial_condition::RCEMIPIIProfile)(params)
```

Comment:
Can you add some more comments / equation references to this function that explains the various equations and the numbers you use.